src/wazuh_db/wdb.h

```diff
@@ -2060,6 +2061,25 @@ wdbc_result wdb_global_assign_agent_group(wdb_t *wdb, int id, cJSON* j_groups, i
  */
 wdbc_result wdb_global_unassign_agent_group(wdb_t *wdb, int id, cJSON* j_groups);
 
+/**
+ * @brief Return the number of groups are assigned to agent_id
+ *
+ * @param wdb The Global struct database.
+ * @param agent_id ID of the agent to get the groups number from.
```

Comment:
Just to be consistent, we could add `[in]` to both parameters

Reply:
Added!